src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnectionPool.cs

```diff
@@ -1553,6 +1553,10 @@ private static bool GetIsWindows7Or2008R2()
                 memberName,                  // method name
                 message);                    // message
 
+        public int Http11ConnectionCount => _associatedConnectionCount;
+
+        public int Http20StreamCount => _http2Connection?.HttpStreamCount ?? 0;
```

Comment:
Could we some how make it more friendly to upcoming [multiple HTTP/2 connection API](https://github.com/dotnet/runtime/issues/35088)?